File: Basics/Prime-up-to-N/prime_up_to_n.py
import math

def print_prime(N):
    for i in range(2, N + 1):
        is_prime = True;

        for j in range(2, int(math.sqrt(i)) + 1):
            if i % j == 0:
                is_prime = False
                break
    
        if is_prime:
            print(f"{i}", end=" ")


n = int(input("Enter the value of N: "))

if n < 2:
    print("No Prime Number in this range.")
else:
    print_prime(n)

File: dsa-cpp-python/Basics/Prime-up-to-N/prime_up_to_n.cpp
#include<iostream>
using namespace std;

void print_prime(int N){
    for(int i = 2; i <= N; i++){

        bool is_prime = true;

        for(int j = 2; j * j <= i; j++){
            if(i % j == 0){
                is_prime = false;
                break;
            }
        }

        if(is_prime){
            cout << i << " ";
        }
    }
}

int main(){
    int n;
    cout << "Enter the value of N: ";
    cin >> n;

    if(n < 2){
        cout << "No Prime Numbers in this range.";
    }
    else{
        print_prime(n);
    }

    return 0;
}